### PR TITLE
Update structure-extensions.md [skip ci]

### PR DESCRIPTION
**Pull request by pana1990**
@klimov-paul can you complete description?

### CODE REVIEW

**Comment by samdark** [comment]
It's too early to mention in in main docs, isn't it?

File: docs/guide/structure-extensions.md [hunk 1]
```diff
@@ -414,6 +414,8 @@ registered on [Packagist](https://packagist.org/) and can be easily installed as
 - [yiisoft/yii2-gii](https://github.com/yiisoft/yii2-gii):
   provides a Web-based code generator that is highly extensible and can be used to quickly generate models,
   forms, modules, CRUD, etc.
+- [yiisoft/yii2-httpclient](https://github.com/yiisoft/yii2-httpclient):
+  provides the HTTP client.
```
Comment (mikehaertl) [inline docs/guide/structure-extensions.md]: Shouldn't it better be "an HTTP client" here? No one mentioned any HTTP client before and there are several imlementations, so it usually gets an indefinite article. There are more "the"s affected on this page I think.

**Comment by pana1990** [comment]
@samdark maybe, wait until you consider it convenient

**Comment by klimov-paul** [comment]
I suppose docs can be updated only after stable release of `yii2-httpclient`.

**Comment by pana1990** [comment]
Of course ;)

**Comment by klimov-paul** [comment]
Resolved by commit b1da1085b4fd7e222dde1cc84a69e242284056d6

**Pull request closed without merging**